internal/datastore/postgres/gc.go

```diff
@@ -68,6 +68,15 @@ func (pgd *pgDatastore) TxIDBefore(ctx context.Context, before time.Time) (datas
 	return postgresRevision{snapshot: snapshot, optionalTxID: value}, nil
 }
 
+func (pgd *pgDatastore) DeleteExpiredRels(ctx context.Context) (int64, error) {
+	return pgd.batchDelete(
+		ctx,
+		tableTuple,
+		gcPKCols,
+		sq.Lt{colExpiration: time.Now().Add(-1 * pgd.gcWindow)},
```

Comment:
Why are we using `time.Now()` here, but for relationships we call `gc.Now()` which calls the datastore? Please use the same function, this is not in the critical path.

Reply:
Changed